receiver/mysqlreceiver/testdata/integration/scripts/setup.sh

```diff
@@ -1,46 +0,0 @@
-#!/usr/bin/env bash
-
-# Copyright The OpenTelemetry Authors
-# SPDX-License-Identifier: Apache-2.0
-
-set -e
-
-USER="otel"
-ROOT_PASS="otel"
-CODE=1
-
-
-setup_permissions() {
-    # NOTE: -pPASSWORD is missing a space on purpose
-    mysql -u root -p"${ROOT_PASS}" -e "GRANT PROCESS ON *.* TO ${USER}" > /dev/null
-    mysql -u root -p"${ROOT_PASS}" -e "GRANT SELECT ON INFORMATION_SCHEMA.INNODB_METRICS TO ${USER}" > /dev/null
-    mysql -u root -p"${ROOT_PASS}" -e "FLUSH PRIVILEGES" > /dev/null
-}
-
-setup_data() {
```

Comment:
It doesn't look like these were replaced anywhere, are they no longer needed?

Reply:
Yeah, they appear to have been extraneous.